### ISSUE TITLE
训练检测时出现，*** Check failure stack trace: ***

### ISSUE THREAD

**Issue by simplew2011** [PaddlePaddle/PaddleOCR#1120]
训练检测时出现，*** Check failure stack trace: ***
而后自动停止，没有其他报错。

将'train_batch_size_per_card': 32, 'test_batch_size_per_card': 32
各自减少一半至16，可开始训练。
此处，因为batch设置的问题导致无法训练，应该报OOM吧，为何没提示，直接结束。
看看能否解。

D:\ProgramData\Anaconda3\envs\tf_1.15\python.exe C:/Users/Administrator/Desktop/wen_ocr/main_ocr_train.py
2020-11-06 17:08:41,110-INFO: {'Global': {'algorithm': 'DB', 'use_gpu': True, 'epoch_num': 100, 'log_smooth_window': 20, 'print_batch_step': 2, 'save_model_dir': './weights\\det', 'save_epoch_step': 5, 'eval_batch_step': [1000, 5000], 'train_batch_size_per_card': 32, 'test_batch_size_per_card': 32, 'image_shape': [3, 640, 640], 'reader_yml': './configs/det/det_db_icdar15_reader_test.yml', 'pretrain_weights': './weights/det/ch_ppocr_server_v1.1_det_train/best_accuracy', 'checkpoints': None, 'save_res_path': './output/det_db/predicts_db.txt', 'save_inference_dir': None}, 'Architecture': {'function': 'ppocr.modeling.architectures.det_model,DetModel'}, 'Backbone': {'function': 'ppocr.modeling.backbones.det_mobilenet_v3,MobileNetV3', 'scale': 0.5, 'model_name': 'large', 'disable_se': True}, 'Head': {'function': 'ppocr.modeling.heads.det_db_head,DBHead', 'model_name': 'large', 'k': 50, 'inner_channels': 96, 'out_channels': 2}, 'Loss': {'function': 'ppocr.modeling.losses.det_db_loss,DBLoss', 'balance_loss': True, 'main_loss_type': 'DiceLoss', 'alpha': 5, 'beta': 10, 'ohem_ratio': 3}, 'Optimizer': {'function': 'ppocr.optimizer,AdamDecay', 'base_lr': 0.001, 'beta1': 0.9, 'beta2': 0.999, 'decay': {'function': 'cosine_decay_warmup', 'step_each_epoch': 16, 'total_epoch': 1200}}, 'PostProcess': {'function': 'ppocr.postprocess.db_postprocess,DBPostProcess', 'thresh': 0.3, 'box_thresh': 0.6, 'max_candidates': 1000, 'unclip_ratio': 1.5}, 'TrainReader': {'reader_function': 'ppocr.data.det.dataset_traversal,TrainReader', 'process_function': 'ppocr.data.det.db_process,DBProcessTrain', 'num_workers': 8, 'img_set_dir': 'D:/Plate_OCR/13/CCPD2019.tar/CCPD2019/', 'label_file_path': 'D:/Plate_OCR/13/CCPD2019.tar/CCPD2019/splits/det_gt_train_1_train.txt'}, 'EvalReader': {'reader_function': 'ppocr.data.det.dataset_traversal,EvalTestReader', 'process_function': 'ppocr.data.det.db_process,DBProcessTest', 'img_set_dir': 'D:/Plate_OCR/13/CCPD2019.tar/CCPD2019/', 'label_file_path': 'D:/Plate_OCR/13/CCPD2019.tar/CCPD2019/splits/det_gt_train_1_val.txt', 'test_image_shape': [736, 1280]}, 'TestReader': {'reader_function': 'ppocr.data.det.dataset_traversal,EvalTestReader', 'process_function': 'ppocr.data.det.db_process,DBProcessTest', 'infer_img': None, 'img_set_dir': 'D:\\Plate_OCR\\13\\CCPD2019.tar\\CCPD2019\\', 'label_file_path': './train_data/icdar2015/text_localization/test_icdar2015_label.txt', 'do_eval': True}}
3 640 640
2020-11-06 17:08:41,915-WARNING: C:\Users\Administrator\Desktop\wen_ocr\ppocr\modeling\heads\det_db_head.py:123
The behavior of expression A - B has been unified with elementwise_sub(X, Y, axis=-1) from Paddle 2.0. If your code works well in the older versions but crashes in this version, try to use elementwise_sub(X, Y, axis=0) instead of A - B. This transitional warning will be dropped in the future.
2020-11-06 17:08:42,322-INFO: If regularizer of a Parameter has been set by 'fluid.ParamAttr' or 'fluid.WeightNormParamAttr' already. The Regularization[L2Decay, regularization_coeff=0.000000] in Optimizer will not take effect, and it will only be applied to other Parameters!
3 640 640
D:/Plate_OCR/13/CCPD2019.tar/CCPD2019/splits/det_gt_train_1_train.txt
2020-11-06 17:08:45,261-INFO: places would be ommited when DataLoader is not iterable
2020-11-06 17:08:48,457-INFO: Loading parameters from ./weights/det/ch_ppocr_server_v1.1_det_train/best_accuracy...
2020-11-06 17:08:49,074-WARNING: variable conv2d_32.w_0 not used
2020-11-06 17:08:49,074-WARNING: variable conv2d_transpose_0.w_0 not used
2020-11-06 17:08:49,074-WARNING: variable conv2d_transpose_1.w_0 not used
2020-11-06 17:08:49,074-WARNING: variable conv2d_transpose_2.w_0 not used
2020-11-06 17:08:49,074-WARNING: variable conv2d_transpose_3.w_0 not used
2020-11-06 17:08:49,075-WARNING: variable conv2_b_attr not used
2020-11-06 17:08:49,108-INFO: Finish initing model from ./weights/det/ch_ppocr_server_v1.1_det_train/best_accuracy
2020-11-06 17:08:49,108-INFO: During the training process, after the 1000th iteration, an evaluation is run every 5000 iterations
multiprocess is not fully compatible with Windows.num_workers will be 1.
*** Check failure stack trace: ***

Process finished with exit code -1073740791 (0xC0000409)

**Comment by tink2123** [comment]
检测模型比较大，且输入图片shape也比较大，windows上如果显存不够的话，建议减小batch_size训练。报错不友好的问题，我们再看一下～

**Comment by grasswolfs** [comment]
由于长期没有收到您的回复，本issue暂时关闭，如果仍然有问题，可以reopen继续咨询。
